llmnrd.c

```diff
@@ -150,6 +150,24 @@ static void hostname_change_handle(char *hostname, size_t maxlen)
 	free(newname);
 }
 
+static bool write_pid_file(void)
+{
+	FILE *fp;
+
+	fp = fopen(PIDFILE, "w");
+	if ( ! fp ) {
```

Comment:
Formatting nit:
```suggestion
	if (!fp) {
```

Reply:
Noted, I wasn't sure which style you used.